lua/cmp_path/init.lua

```diff
@@ -145,6 +145,14 @@ source._candidates = function(_, params, dirname, offset, callback)
     accept = accept or include_hidden
     accept = accept or name:sub(1, 1) ~= '.'
 
+    if type == nil then
```

Comment:
Please avoid the duplicate fs_stat call.

https://github.com/hrsh7th/cmp-path/pull/14/files#diff-09e74c4a53886ca79ae4d4022259a258135b3e44a39986cc9128f63633dfd2f2R166

Reply:
Done